data/coverage/2020.yml

```
@@ -31,6 +31,12 @@ april:
   - type: video
     url: https://assets.lrug.org/videos/2020/april/joel-chippindale-how-to-take-control-of-your-code-quality-lrug-apr-2020.mp4
     title: LRUG April 2020 - Joel Chippindale - How to take control of code quality
+  - type: slides
+    url: https://blog.mocoso.co.uk/assets/take-control-of-code-quality/take-control-of-code-quality--lrug-apr-2020.pdf
+    title: 'How to take control of code quality'
+  - type: link
```

I think transcript (or write-up) might be more relevant here:
```suggestion
  - type: transcript
```
Only because some future version of the coverage stuff _might_ differentiate links by type in a more interesting manner.  Totally minor though and I'll fix this up myself.  Thanks for the links in the first place @mocoso